### PR TITLE
:lock: default saas-access for user, admin

### PR DESCRIPTION
**Pull request by fedoranvar**

### CODE REVIEW

File: saas/security/saas_security.xml [hunk 1]
```diff
@@ -8,13 +8,14 @@
   <record model="res.groups" id="group_user">
     <field name="name">User</field>
     <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
-    <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    <field name="users" eval="[(4, ref('base.user_demo'))]"/>
```
Comment (trojikman) [inline saas/security/saas_security.xml]: I think it's better to separate user_demo like this: https://github.com/odoo/odoo/blob/12.0/addons/lunch/data/lunch_demo.xml#L6
We have demo directory for it. Here, just as before, use user_root

File: saas/security/saas_security.xml [hunk 2]
```diff
@@ -8,13 +8,14 @@
...
     <field name="category_id" ref="saas.module_management"/>
   </record>
 
   <record model="res.groups" id="group_manager">
     <field name="name">Manager</field>
     <field name="implied_ids" eval="[(4, ref('saas.group_user'))]"/>
+    <field name="users" eval="[(4, ref('base.user_admin'))]"/>
```
Comment (trojikman) [inline saas/security/saas_security.xml]: Complete this entry with this: (4, ref('base.user_root'))

**Review by trojikman — CHANGES_REQUESTED** [review]

**Review by yelizariev — CHANGES_REQUESTED** [review]
you reference to demo data outside of demo files

File: saas/security/saas_security.xml (hunk 2 above)
Comment (fedoranvar) [reply saas/security/saas_security.xml]: I'm not quite sure, but if we add (4, ref('base.user_root')) to (4, ref('base.user_admin')), then average users will have saas-manager-rights

File: saas/demo/saas_template_demo.xml [hunk 3]
```diff
@@ -13,4 +13,7 @@
         <field name="template_id" eval="ref('saas_template')"/>
         <field name="operator_id" eval="ref('local_operator')"/>
     </record>
+    <record model="res.groups" id="saas.group_user">
+      <field name="users" eval="[(4, ref('base.user_root')),(4, ref('base.user_demo'))]"/>
+    </record>
```
Comment (yelizariev) [inline saas/demo/saas_template_demo.xml]: Use another file. You put security stuff in file named "template"

File: saas/security/saas_security.xml (hunk 2 above)
Comment (yelizariev) [reply saas/security/saas_security.xml]: You need to add both values: for superuser and for admin-admin

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: saas/security/saas_security.xml [hunk 4]
```diff
@@ -15,6 +17,7 @@
   <record model="res.groups" id="group_manager">
     <field name="name">Manager</field>
     <field name="implied_ids" eval="[(4, ref('saas.group_user'))]"/>
+    <field name="users" eval="[(4, ref('base.user_admin')),(4, ref('base.group_system')) ]"/>
```
Comment (yelizariev) [inline saas/security/saas_security.xml]: group_system is not a user

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: saas/security/saas_security.xml [hunk 5]
```diff
@@ -8,13 +10,14 @@
   <record model="res.groups" id="group_user">
     <field name="name">User</field>
     <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
-    <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    <field name="users" eval="[(4, ref('base.default_user'))]"/>
```
Comment (yelizariev) [inline saas/security/saas_security.xml]: You shall not update user template

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**